### DISCUSSION TITLE
Welcome to AutoFetcher-IG-Stories-to-GDrive Discussions!

### DISCUSSION THREAD

**Discussion by chriskyfung** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cerberus1730** [comment]
Hello,
I have problem with script.
When I choose Run, I get notifs about:
"The number of rows in the range must be at least 1"
I configure all basic settings and run script when I be log in Instagram account, but script not work...
Can you help me?

**Reply by chriskyfung** [reply]
Hi,
What did you put on the subscriptions page?
Could you upload a screenshot?

**Reply by cerberus1730** [reply]
Hello,
I put one subscription page. I try fix this by View->Freeze->1 row, but no work too.
![badef98279f01639e23898c4bc480bc4](https://user-images.githubusercontent.com/96347274/147112220-88363367-fbdd-4f0c-9877-77b00d2e5bb4.png)

**Reply by ghost** [reply]
Try adding some logs.
![IMG_20211222_232352](https://user-images.githubusercontent.com/94428405/147115507-eae5b1ab-a35e-4664-8b5c-084591d1cc54.jpg)

**Reply by chriskyfung** [reply]
I reproduced the problem when no records on the **Logs** page.

It can be fixed by simply making the second row have a non-blank cell, such as inserting a checkbox to the **Selected** column

**Reply by cerberus1730** [reply]
> Try adding some logs. ![IMG_20211222_232352](https://user-images.githubusercontent.com/94428405/147115507-eae5b1ab-a35e-4664-8b5c-084591d1cc54.jpg)

Thank you very much, now it's work

**Comment by Moni0516** [comment]
Hello,

first of all sry for my englisch :D.

everything is working fine except of Destination Folder ID. I put the right id of the folder in the cell but the file is saveing in the Google Drive root folder. Can someone tell me what i am doing wrong ?

**Reply by chriskyfung** [reply]
A pending feature

**Reply by Moni0516** [reply]
thank you

**Reply by chriskyfung** [reply]
See #20

**Reply by chriskyfung** [reply]
Support **Destination Folder ID** from Build2023.02.14a (Library V8) 🐱‍🏍

**Comment by Moni0516** [comment]
![error](https://user-images.githubusercontent.com/104869460/166641130-c5090647-6b65-487d-bc3e-8c065a1c835e.PNG)

hello, i have this problem today. i cant add new profiles

**Reply by chriskyfung** [reply]
Move to #50
